src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -322,6 +322,8 @@
                 This consent is valid <strong>until {{filing.expiry}} at 12:01 am Pacific time</strong>.
               </p>
               <p class="mt-4">{{filing.comment}}</p>
+              <p v-if="filing.fileNumber" class="mt-4 mb-0">Court Order Number: {{filing.fileNumber}}</p>
+              <p v-if="filing.planOfArrangement" class="mt-0">{{filing.planOfArrangement}}</p>
```

Comment:
These might not be displayed correctly but the uxpin has not been updated. This can be fixed later.